recipes/pluggy/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "pluggy" %}
+{% set version = "0.4.0" %}
+{% set sha256 = "dd841b5d290b252cf645f75f3bd37ceecfa0f36394ab313e4f785fe68a4081a4" %}
+
+package:
+  name: pluggy
```

Comment:
Please use the jinja templating variables:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L13-L14